### PR TITLE
fix(atomic): remove max-with for breadcrumbs and ellipse the content

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-616
Screen Shot 2021-04-15 at 3 13 38 PM

### CODE REVIEW

File: packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx [hunk 1]
```diff
@@ -23,14 +23,9 @@ import dayjs from 'dayjs';
 /**
  * A component that creates breadcrumbs that display the currently active facet values
  *
- * @part breadcrumbs - Container for all types of breadcrumbs
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx:26]: I found that there were too many parts here, leading to confusion. I left the crucial ones, but I could add back more

File: packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx [hunk 2]
```diff
@@ -81,24 +76,20 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
   ) {
     return (
       <button
-        part="breadcrumb-button"
-        class="text-on-background-variant breadcrumb-button flex items-center hover:text-primary-variant w-full"
+        part="breadcrumb"
+        class="inline-grid grid-flow-col"
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx:80]: this is the key line, too me a while to figure this one out 😩

File: packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx [hunk 3]
```diff
@@ -276,20 +271,21 @@ export class AtomicBreadcrumbManager implements InitializableComponent {
...
-        </button>
-      </li>
+    if (collapsedBreadcrumbNumber < 1) {
+      return;
+    }
+
+    return this.getBreadcrumbValueWrapper(
+      <button
+        part="breadcrumb"
+        // TODO: localize
```
Comment (ThibodeauJF) [inline packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx:281]: https://coveord.atlassian.net/browse/KIT-624

**Review by samisayegh — APPROVED** [review]

**Comment by olamothe** [comment]
Based on the screenshot though... it's still kind of a eye sore ? The text looks mis-aligned and is not natural to read.
IMO, the perfect thing would be : 

```
// Break word: all. but ensure that the last item on a line is at least X pixel long 
// so that we can "guess" what it is pretty correctly even with the ellipsis, since there is enough leading characters
Authors: [amoreau X] [Lily Chan X] [LilyC X] [OHBOYOHBOYOHBOY... X]
         [HEYHEYHEYHEY .... X]


// Meaning: we don't want this to happen, where the last item is so small that it's impossible to know what it is
Authors: [amoreau X] [Lily Chan X] [LilyC X] [OH... X]
```

**Pull request merged**